src/components/Observations/GridItem.js

```diff
@@ -1,118 +1,114 @@
 // @flow
 
 import classnames from "classnames";
-import { Image, Pressable, View } from "components/styledComponents";
+import DisplayTaxonName from "components/DisplayTaxonName";
+import { ImageBackground, Pressable, View } from "components/styledComponents";
 import { t } from "i18next";
 import type { Node } from "react";
 import React from "react";
+import LinearGradient from "react-native-linear-gradient";
 import IconMaterial from "react-native-vector-icons/MaterialIcons";
 import Observation from "realmModels/Observation";
 import Photo from "realmModels/Photo";
 import colors from "styles/tailwindColors";
 
-import ObsCardDetails from "./ObsCardDetails";
 import ObsStatus from "./ObsStatus";
 import UploadButton from "./UploadButton";
 
 type Props = {
-  // position of this item in a list of items; not ideal, but it allows us to
-  // style grids appropriately
-  index?: number,
-  item: Object,
+  observation: Object,
   handlePress: Function,
-  // Number of columns in the grid; we need this to set the margins correctly
-  numColumns?: number,
   uri?: string,
 };
 
 const GridItem = ( {
```

Comment:
Let's rename this component and file to `ObsGridItem` since it is really only for rendering observations in a grid.